From 0cb952c2f6f9389f6abaef92ed0f06cd59eeb95f Mon Sep 17 00:00:00 2001
From: Jianzheng Zhou <jianzheng.zhou@freescale.com>
Date: Sat, 27 Apr 2013 13:44:54 +0800
Subject: [PATCH 192/304] ENGR00260719 Wifi display:fix device node permission
 issue

Change media user has permission to operate /dev/mxc_ipu.
or will report following error:
I/OMXPlayer( 2285): LEVEL: 1 FUNCTION: PreProcessIPUInit LINE: 826
I/OMXPlayer( 2285): Open /dev/mxc_ipu failed.
I/OMXPlayer( 2285): LEVEL: 1 FUNCTION: FilterOneBuffer LINE: 2227
I/OMXPlayer( 2285): init pre-process failure !

Signed-off-by: Jianzheng Zhou <jianzheng.zhou@freescale.com>
---
 imx6/etc/ueventd.freescale.rc | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/imx6/etc/ueventd.freescale.rc b/imx6/etc/ueventd.freescale.rc
index d84f19f..9099a66 100644
--- a/imx6/etc/ueventd.freescale.rc
+++ b/imx6/etc/ueventd.freescale.rc
@@ -8,7 +8,7 @@
 /dev/video1               0660   system     camera
 /dev/video2               0660   system     camera
 /dev/video16              0660   system     graphics
-/dev/mxc_ipu              0660   graphics   system
+/dev/mxc_ipu              0660   media      system
 /dev/mxc_iim              0664   system     media
 /dev/mxs_viim             0664   system     media
 /dev/fsl_cache            0666   system     graphics
-- 
1.8.0

